Chap_API_Sync_Access.tex

```diff
@@ -180,12 +431,15 @@ \section{\code{PMIx_Get}}
 %%%%
 \format
 
-\copySignature{PMIx_Get}{1.0}{
-pmix_status_t \\
-PMIx_Get(const pmix_proc_t *proc, const pmix_key_t key, \\
-\hspace*{9\sigspace}const pmix_info_t info[], size_t ninfo, \\
-\hspace*{9\sigspace}pmix_value_t **val);
-}
+\versionMarker{1.0}
+\cspecificstart
+\begin{codepar}
+pmix_status_t
+PMIx_Get(const pmix_proc_t *proc, const pmix_key_t key,
+         const pmix_info_t info[], size_t ninfo,
+         pmix_value_t **val);
+\end{codepar}
+\cspecificend
```

Comment:
Editorial: I think we need to adjust this to use the `\copySignature` syntax used to support the Use Case appendix.